Cargo.toml

```diff
@@ -2,11 +2,16 @@
 name = "projectm_sdl"
 version = "0.1.0"
 edition = "2021"
+rust-version = "1.68.2"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 libc = "*"
-sdl2 = "0.35.2"
-gl = "0.14.0"
-projectm-rs = "^1.0.4"
+sdl2 = "0.35"
```

Comment:
is this a downgrade of sdl2 from 0.35.2?

Reply:
nope. it will automatically pull in 0.35.2. if it there is 0.35.3 it will auto pull 0.35.3